src/api/entityResolution/index.ts

```diff
@@ -148,6 +150,24 @@ export class EntityResolutionAPI extends Request {
     });
   }
 
+  /**
+   * Stop an ingestion task on a given data-source.
+   */
+  stopIngestion(this: Request<IngestionStatus>, params: StopEntityResolutionTaskParams) {
+    return this.request({
+      errors: [
+        UNAUTHORIZED,
+        FORBIDDEN,
+        DATA_SOURCE_UNAVAILABLE,
+        NOT_SUPPORTED,
```

Comment:
when does this error happen?

Reply:
This error is just a contingency in case the `stop` API is called on a datasource that does not support ER